src/dialect/postgres/postgres-query-compiler.ts

```diff
@@ -5,6 +5,18 @@ export class PostgresQueryCompiler extends DefaultQueryCompiler {
     return '$' + this.numParameters
   }
 
+  protected override getLeftExplainOptionsWrapper(): string {
```

Comment:
The default query compiler is supposed to be for postgres. Use these as the defaults in `DefaultQueryCompiler`

Reply:
We should probably remove all these overrides here to make that more obvious.

Reply:
> We should probably remove all these overrides here to make that more obvious.

Good idea! done!